### PR TITLE
bugfix so apps will save again

### PR DESCRIPTION
**Pull request by jenbeckett**

### CODE REVIEW

**Review by jgstorey — APPROVED** [review]
Reviewed approved.

File: frontend/src/services/applicationService.js [hunk 1]
```diff
@@ -98,6 +98,32 @@ export default {
     }
   },
 
+  async getSupplementaryApplicationsForForm(applicationId) {
+    try {
+      if (!applicationId) return
+      const url = `${ApiRoutes.SUPPLEMENTARY_APPLICATIONS}/${applicationId}`
+
+      const response = await ApiService.apiAxios.get(url)
+      //clean up the fields I don't need
```
Comment (weskubo-cgi) [inline frontend/src/services/applicationService.js:107]: Would it make more sense to just include the fields you do need? What if I new field is added? Will it cause issues if not deleted here?

**Review by weskubo-cgi — CHANGES_REQUESTED** [review]

File: frontend/src/services/applicationService.js (hunk 1 above)
Comment (jenbeckett) [reply frontend/src/services/applicationService.js:105]: nope -it was leftover from the other code - it's gone now

Comment (weskubo-cgi) [reply frontend/src/services/applicationService.js:105]: Discussed with Jen and there are more fields retained than deleted so the current approach makes sense.

**Review by weskubo-cgi — APPROVED** [review]

**Pull request merged**